package com.futmatch.convocatoria_ms.exceptions;

public class ConvocatoriaNotFoundException extends RuntimeException {
    public ConvocatoriaNotFoundException(String message) {
        super(message);
    }
}
